resync/knowledge/memory/chat_memory.py

```
@@ -0,0 +1,254 @@
+        user_id: ID do usuário (para personalização)
+        role: 'user' | 'assistant'
+        content: Texto da mensagem
+        expires_at: Data de expiração (TTL GDPR)
+        metadata: Dados adicionais opcionais
+        created_at: Timestamp de criação
+    """
+    session_id: str
+    user_id: str
+    role: str
+    content: str
+    expires_at: datetime = field(default_factory=lambda: datetime.utcnow() + timedelta(days=CHAT_MEMORY_TTL_DAYS))
```

Ruff (0.15.0)

[warning] `datetime.datetime.utcnow()` used

(DTZ003)